docker-compose.yml

```diff
@@ -35,7 +35,10 @@ services:
       DATALAND_SKYMINDERCLIENT_USER: ${DATALAND_SKYMINDERCLIENT_USER}
   frontend:
     build:
-      context: ./dataland-frontend
+      context: ./
+      dockerfile: ./dataland-frontend/Dockerfile
+    volumes:
+      - ./dataland-frontend/default.conf:/etc/nginx/conf.d/default.conf
```

Comment:
wenn wir diese config hier brauchen, brauchen wir sie wohl auch auf preview. Im docker-compose file in `deployment` fehlt das aber.

Reply:
ok